pkg/resources/sku/sku.go

```diff
@@ -172,6 +176,9 @@ func (p AProductConfig) Configure(obj metav1.Object) error {
 		return errors.New(fmt.Sprintf("sku configuration can only be applied to Deployments, StatefulSets or Deployment Configs, found %s", reflect.TypeOf(obj)))
 	}
 
+	if replicas == nil {
+		*replicas = defaultReplicas
```

Comment:
Shouldn't it be `replicas = &defaultReplicas`? Now we are "if the pointer is nill, store in that nill location my value".  IT should be "you have a nill? here is an address you need"